js/ui/text_box/ui.text_editor.base.js

```diff
@@ -432,6 +443,27 @@ const TextEditorBase = Editor.inherit({
         this._input().prop('spellcheck', this.option('spellcheck'));
     },
 
+    _renderLabel: function(options) {
+        if(this._$label) {
+            this._$label.remove();
+            this._$label = null;
+        }
+
+        this.$element().removeClass(TEXTEDITOR_WITH_LABEL_CLASS);
+        this.$element().removeClass(TEXTEDITOR_WITH_FLOATING_LABEL_CLASS);
```

Comment:
this.$element()
    .removeClass(TEXTEDITOR_WITH_LABEL_CLASS)
    .removeClass(TEXTEDITOR_WITH_FLOATING_LABEL_CLASS);